grammars.yml

```diff
@@ -811,6 +811,10 @@ vendor/grammars/lua.tmbundle:
 - source.lua
 vendor/grammars/m3:
 - source.modula-3
+vendor/grammars/magik-vscode:
+- source.magik
+- text.definition
+- text.message
```

Comment:
These `text` ones seem erroneous - only `source.magik` is used.

Reply:
Thank you for your feedback! I agree with this statement, however was unsure if I should modify grammars.yml as this was automatically generated by the scripts.

Reply:
Do not manually modify this file. It's maintained by a script and should remain that way. The script will always find all grammars in the repo and add them to this file so there's nothing wrong with this.